packages/ssx-sdk/src/modules/Encryption.ts

```diff
@@ -1,7 +1,28 @@
+import * as jose from 'jose';
 import { IUserAuthorization } from './UserAuthorization';
+
+/**
+ * The Encryption module handles the encryption and decryption of data.
+ * It expects a Blob as input to be encrypted and returns a Blob as output when
+ * decrypted.
+ *
+ * The format encrypted state depends on the implementation.
+ */
 interface IEncryption {
-  // encrypt
-  // decrypt
+  /**
+   * Encrypts the data. The data is expected to be a Blob.
+   * The output type depends on the implementation.
+   * @param data - The data to encrypt.
+   * @returns The encrypted data.
+   */
+  encrypt: (data: Blob) => Promise<any>;
+  /**
+   * Decrypts the data. The encrypted type depends on the implementation. The
+   * output returned is a Blob.
+   * @param encrypted - The encrypted data to decrypt.
+   * @returns - The decrypted data as a blob
+   */
+  decrypt: (encrypted: any) => Promise<Blob>;
```

Comment:
```suggestion
interface IEncryption<E> {
  /**
   * Encrypts the data. The data is expected to be a Blob.
   * The output type depends on the implementation.
   * @param data - The data to encrypt.
   * @returns The encrypted data.
   */
  encrypt: (data: Blob) => Promise<E>;
  /**
   * Decrypts the data. The encrypted type depends on the implementation. The
   * output returned is a Blob.
   * @param encrypted - The encrypted data to decrypt.
   * @returns - The decrypted data as a blob
   */
  decrypt: (encrypted: E) => Promise<Blob>;
}
```
For the sake of types, I'd suggest this interface have a generic param representing the cipher text. For example, signature encryption would `implements Encryption<JWE>` (or whatever the JWE type is called). Mostly because `any` here would make it possible to pass in a different type to `decrypt` than the one returned by `encrypt` and in general will make it hard to figure out what's up by looking at types

Reply:
Can make this change